src/fhirRequest.py

```diff
@@ -0,0 +1,52 @@
+import datetime
+from constants import REFRESH_TOKEN_ENDPOINT, SDK_HEADER, SDK_HEADER_KEY
+import requests
+from requests.adapters import HTTPAdapter, Retry
+
+
+def fhirRequest(bb, config):
+    auth_token = config["auth_token"]
+    new_auth_token = handle_expired(bb, auth_token)
+
+    if new_auth_token is not None:
+        auth_token = new_auth_token
+
+    retry_config = Retry(
+        total=3, backoff_factor=0.2, status_forcelist=[500, 502, 503, 504]
```

Comment:
I think we should add 429 here for "rate limit exceeded".

The backoff_factor seems low?  Especially for the back end related errors that are resource related and might take a little time for a repeat call to be successful.  

Found these examples of retry timing per backoff_factor:

1 second the successive sleeps will be 0.5, 1, 2, 4, 8, 16, 32, 64, 128, 256.
2 seconds - 1, 2, 4, 8, 16, 32, 64, 128, 256, 512
10 seconds - 5, 10, 20, 40, 80, 160, 320, 640, 1280, 2560

Thinking maybe 5 or 10 might be a better value with 3x retries.